src/gConsent/constants.ts

```diff
@@ -98,6 +98,7 @@ export const ACCESS_CREDENTIAL_TYPE = new Set([
   CREDENTIAL_TYPE_ACCESS_REQUEST,
   CREDENTIAL_TYPE_ACCESS_GRANT,
   CREDENTIAL_TYPE_ACCESS_DENIAL,
+  "vc:SolidAccessDenial",
```

Comment:
Denied grants currently use prefixes. So the client side filtering needs to check for this format.